account_invoice_analytic_search/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License AGPL-3
+
+Invoice Analytic Search
+=======================
+
+Organizations often require to quickly find the invoices associated to a
+project or to an analytic account, searching by it's code, name or
+project/account manager. This module introduces the possibility search
+customer or supplier invoices by analytic account or by project manager.
+
+Installation
+============
+
+* The module automatically takes care of its dependencies and is ready for use after the installation
+
+Credits
+=======
```

Comment:
Please adjust this to the OCA guidelines:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Also remove the icon